### PR TITLE
DRAFT: RDBMS Section for Concepts

### PR DESCRIPTION
**Pull request by timsehn**

### CODE REVIEW

**Comment by VinaiRachakonda** [comment]
I like it!

File: content/concepts/dolt/rdbms/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: Relational Database Management System
+---
+
+Dolt is can be used as a Relational Database Management System or RDMS. Dolt ships with [a MySQL compatible server](./server.md) built in, accessed via the [`dolt sql-server` command](../../../reference/cli.md#dolt-sql-server).
```
Comment (max-hoffman) [inline content/concepts/dolt/rdbms/README.md]:
```suggestion
Dolt can be used as a Relational Database Management System or RDMS. Dolt ships with [a MySQL compatible server](./server.md) built in, accessed via the [`dolt sql-server` command](../../../reference/cli.md#dolt-sql-server).
```

File: content/concepts/dolt/rdbms/replication.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+---
+title: Replication
+---
+
+# Replication
+
+## What is a Replication?
```
Comment (fulghum) [inline content/concepts/dolt/rdbms/replication.md]:
```suggestion
## What is Replication?
```

File: content/concepts/dolt/rdbms/replication.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+Replication is the ability for an RDBMS to synchronize a master server with one or more read replicas. In this configuration, the master database serves reads and writes while the replicas serve just reads.
+
+## How to use Replication
+
+Replication is used for disaster recovery and to increase read throughput by distributing read load.
+
+For the disaster recovery, if your master server goes offline, your database can still serve read traffic from its replicas. Often a manual or automated process can elect and configure a replica to be the master instance limiting downtime.
```
Comment (fulghum) [inline content/concepts/dolt/rdbms/replication.md]:
```suggestion
For disaster recovery, if your master server goes offline, your database can still serve read traffic from its replicas. Often a manual or automated process can elect and configure a replica to be the master instance limiting downtime.
```

File: content/concepts/dolt/rdbms/replication.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+To increase read throughput, multiple replicas can be used to scale reads horizontally. If you have N replicas and your master still takes reads, each read replica serves 1/N+1 percent of the read traffic.
```
Comment (fulghum) [inline content/concepts/dolt/rdbms/replication.md]: Should we mention anything about how to use replicas from your application? Maybe just acknowledging that application code need to be aware of the replicas in order to route requests there, so people don't think replication will reroute requests automatically?

File: content/concepts/dolt/rdbms/replication.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+## Difference between MySQL Replication and Dolt Replication
```
Comment (fulghum) [inline content/concepts/dolt/rdbms/replication.md]:
```suggestion
## Differences between MySQL Replication and Dolt Replication
```

**Review by fulghum — APPROVED** [review]
Content looks good. There's some duplication with the existing "SQL Reference" -> "Running the Server" section. Are you thinking about combining this with that content?

File: content/concepts/dolt/rdbms/replication.md (hunk 4 above)
Comment (timsehn) [reply content/concepts/dolt/rdbms/replication.md]: Good idea.

**Comment by timsehn** [comment]
@fulghum I'm thinking about rewriting those to be in depth How To's. They are not great docs right now.

**Pull request merged**